config/locales/ru.yml

```diff
@@ -0,0 +1,269 @@
+---
+ru:
+  activerecord:
+    errors:
+      messages:
+        record_invalid: 'Возникли ошибки: %{errors}'
+        restrict_dependent_destroy:
+          has_one: 'Невозможно удалить запись, так как существует зависимость: %{record}'
+          has_many: 'Невозможно удалить запись, так как существуют зависимости: %{record}'
+  date:
+    abbr_day_names:
+    - Вс
+    - Пн
+    - Вт
+    - Ср
+    - Чт
+    - Пт
+    - Сб
+    abbr_month_names:
```

Comment:
just noting - comment below about month names applies here for these abbreviations too, but people would most likely be abbreviating a month if using it with a day anyway

Reply:
Ok, so let's keep it too.